content/integrations/lambdatest-integration.md

```diff
@@ -0,0 +1,52 @@
+---
+title: LambdaTest integration
+description: How to integrate your workflows with LambdaTest using codemagic.yaml
+weight: 6
+---
+
+**LambdaTest** is a cloud-based cross browser testing tool that allows you to perform cross browser testing for mobile applications. You can perform both manual and automated cross browser testing using LambdaTest. More information can be found here. 
+
+## Create a LambdaTest account
+
+Registering with LambdaTest is required in order to be able to get the username and access token. You can sign up for free [here](https://www.lambdatest.com/).
+
+## Configuring Real Time and App Automation with Codemagic
+
+**LambdaTest** offers two testing environments, **Real Time** and **App Automation**. Applications can be submitted to both testing environments through Codemagic using a cURL request. In order to configure them correctly, you will need two environment variables: **username** and **access token**. They can be found in the **LambdaTest UI** with your account. 
+
+## Real Time and App Automation
+
+You can test your **.ipa** and **.apk** directly on real devices by submitting them to the **LambdaTest** environment via a cURL request:
+
+```
+-Scripts:
+    - name: Submitting app to LambdaTest:
+    script: |
+        curl --location --request POST 'https://manual-api.lambdatest.com/app/upload/realDevice' --header 'Authorization: Basic $LAMBDATEST' --form 'name="lambda1"' --form 'appFile=@"app/build/outputs/apk/release/app-release.apk"'
+```
+
+**$LAMBDATEST** environment variable is a custom name and can be named to anything preferred. More info can ber found [here](https://docs.codemagic.io/variables/environment-variable-groups/) about how to set up variable groups with Codemagic. The environment variable is the base64 encoded form of your username and access token. They need to be encoded with the following format [here](https://mixedanalytics.com/knowledge-base/api-connector-encode-credentials-to-base-64/):
```

Comment:
This section here is confusing for me. In the previous section, you mentioned that 2 environment variables are needed but here you are mentioning just one.
Also I'm not sure why base64-encoding is needed, is that so as to not expose the values during the build?

Reply:
I might need to change "two env vars" to "two pieces of data" because Lambdatest requires it be base64 encoded together like username:accesstoken

Reply:
Yes, most probably it is because of not exposing them.